hknweb/settings/common.py

```diff
@@ -194,7 +194,17 @@
 
 # Constants for backend code
 
+# user groups
 CAND_GROUP = 'candidate'
 OFFICER_GROUP = 'officer'
 
+# default hard-coded event types for candidate semester
+# NOTE: these strings are also hard-coded in candidate/index.html
+MANDATORY_EVENT = 'mandatory'
```

Comment:
Was the change from `mandatory_meetings` to `mandatory` intentional?

Reply:
Yes, it made more sense to me because then it's consistent with what they call it on the user end, but I guess calling it mandatory meetings is more accurate